src/main/java/uk/gov/hmcts/divorce/document/print/documentpack/DocumentPackInfo.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.divorce.document.print.documentpack;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.divorce.document.model.DocumentType;
+
+import java.util.Map;
+import java.util.Optional;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class DocumentPackInfo {
```

Comment:
this is probably a good candidate to be a record

Reply:
done